src/App/components/InputGroup/index.js

```diff
@@ -137,7 +133,6 @@ InputGroup.propTypes = {
     disabled: PropTypes.bool,
     readOnly: PropTypes.bool,
     label: PropTypes.string,
```

Comment:
review which propTypes should be removed, and which ones added:
- why removing `validationState` ?
- please remove the ones that were removed from the InputGroup params (e.g. `expandingHintTitle`,`expandingHintContent`, `expanderId`, ... )
- please add the ones that got added to the InputGroup params (e.g. `placeholder`,`success`, `error`)